src/coreclr/src/jit/jitconfigvalues.h

```diff
@@ -22,6 +22,7 @@ CONFIG_INTEGER(BreakOnDumpToken, W("BreakOnDumpToken"), 0xffffffff) // Breaks wh
 CONFIG_INTEGER(DebugBreakOnVerificationFailure, W("DebugBreakOnVerificationFailure"), 0) // Halts the jit on
                                                                                          // verification failure
 CONFIG_INTEGER(DiffableDasm, W("JitDiffableDasm"), 0)            // Make the disassembly diff-able
+CONFIG_INTEGER(DasmWithAddress, W("JitDasmWithAddress"), 0)      // Print the process address next to each instruction of the disassembly
```

Comment:
Can we make the config name exactly the same as the COMPlus variable name? Then you don't need to remember the mapping in your head when looking at the code. (Yes, there are some existing exceptions, which are probably historical)